db/schema.rb

```diff
@@ -124,8 +124,17 @@
     t.boolean "notify_email"
     t.string "phone_number"
     t.string "name", default: "", null: false
+    t.string "confirmation_token"
```

Comment:
Looks like the confirmation schema leaked.  In the interest of keeping a consistent schema, go ahead and re-generate the schema with `rails db:drop db:create db:migrate` and commit the newly generated schema file.